lib/src/interactions.dart

```diff
@@ -263,10 +269,27 @@ class Interactions implements IInteractions {
   @override
   Future<void> deleteGlobalCommand(Snowflake commandId) => interactionsEndpoints.deleteGlobalCommand(client.appId, commandId);
 
+  /// Deletes all global commands
+  @override
+  Future<void> deleteGlobalCommands() async => interactionsEndpoints.bulkOverrideGlobalCommands(client.appId, []);
+
   /// Deletes guild command
   @override
   Future<void> deleteGuildCommand(Snowflake commandId, Snowflake guildId) => interactionsEndpoints.deleteGuildCommand(client.appId, commandId, guildId);
 
+  /// Deletes all guild commands for the specified guilds
+  @override
+  Future<void> deleteGuildCommands(List<Snowflake> guildIds) async {
+    List<Future<int>> futures = [];
+
+    for (final guildId in guildIds) {
+      futures.add(interactionsEndpoints.bulkOverrideGuildCommands(client.appId, guildId, []).length);
+    }
+
+    // Wait for all overrides to complete
+    await Future.wait(futures);
```

Comment:
awaiting one by one would be more readable approach

Reply:
It is more readable but less efficient since the requests aren't executed in parallel.

Reply:
```suggestion
    await Future.wait(
      guildIds.map((guildId) => interactionsEndpoints.bulkOverrideGuildCommands(client.appId, guildId, []))
    );
```

that would be better then

Reply:
Correct, I forgot about map.